services/order.py

```diff
@@ -0,0 +1,52 @@
+from datetime import datetime
+from django.db import transaction
+from db.models import Order, User, MovieSession, Ticket
+
+
+@transaction.atomic
+def create_order(
+    tickets: list[dict], username: str, date: datetime = None

```

Comment:
Using `datetime` as a default argument value is not recommended because it is mutable. Consider using `None` as the default value and then initializing it inside the function if needed. For example, use `date: datetime = None` and then `date = date or datetime.now()` inside the function.